aws_codeseeder/services/cfn.py

```diff
@@ -174,7 +174,7 @@ def does_stack_exist(
         if len(resp["Stacks"]) < 1:
             return (False, {})
         else:
-            output = {o["OutputKey"]: o["OutputValue"] for o in resp["Stacks"][0].get("Outputs", [])}
+            output = {o["OutputKey"]: o.get("OutputValue", o.get("ExportName")) for o in resp["Stacks"][0].get("Outputs", [])}
```

Comment:
<img width="1505" alt="image" src="https://github.com/awslabs/aws-codeseeder/assets/32723820/95322f40-8d9a-48aa-8345-f7cf9cb49943">

Reply:
I am not sure what this corresponds to.  I don't see a reference to this output in SF (https://github.com/awslabs/seed-farmer/pull/580) nor have I seen this stacktrace before.